src/main/groovy/com/ullink/gradle/nunit/NUnit3Mixins.groovy

```diff
@@ -3,17 +3,42 @@ package com.ullink.gradle.nunit
 import org.gradle.api.GradleException
 
 class NUnit3Mixins {
-
-    def where
-
     def resultFormat
 
     // Deprecated
     void setRun(def run) {
-        logDeprecatedParameters('run', 'test')
+        logDeprecatedParameters('run', 'where')
         this.setTest(run)
     }
 
+    def setTestInternal(def test, def where, def input) {
```

Comment:
testWrapper and whereWrapper for the parameter names would make it a bit clearer.